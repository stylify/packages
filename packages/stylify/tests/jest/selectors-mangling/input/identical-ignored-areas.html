<!--
stylify-components
	link: 'color:blue'
/stylify-components
-->
<div class="menu-link link"></div>
<div class="menu-link link"></div>

<div class="menu__link link"></div>
<div class="menu__link link"></div>
