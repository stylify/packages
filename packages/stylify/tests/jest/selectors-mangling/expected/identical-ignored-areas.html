<!--
stylify-components
	link: 'color:blue'
/stylify-components
-->
<div class="menu-link a"></div>
<div class="menu-link a"></div>

<div class="menu__link a"></div>
<div class="menu__link a"></div>
